src/planscape/planning/views.py

```diff
@@ -689,6 +690,16 @@ def update_scenario(request: Request) -> Response:
             scenario.name = new_name
             is_dirty = True
 
+        if "status" in body:
```

Comment:
shouldn't we be checking user permissions here? maybe not in scope of this particular pr?

Reply:
nevermind just saw that we do this above :D